cluster-autoscaler/cloudprovider/builder/builder_all.go

```diff
@@ -75,7 +75,7 @@ var AvailableCloudProviders = []string{
 	cloudprovider.LinodeProviderName,
 	cloudprovider.BizflyCloudProviderName,
 	cloudprovider.BrightboxProviderName,
-	cloudprovider.PacketProviderName,
```

Comment:
I think we still need PacketProviderName here for the provider to legacy provider to be used

Reply:
I see - AvailableCloudProviders is only used for command line documentation, it is not used for command line validation.  This should be fine then, as it directs users to use the new provider name but does not prevent usage of the legacy name.

Reply:
I think we have to change the name here also, to maintain uniformity.

Reply:
Yes, already changed and added equinix provider name `cloudprovider.EquinixMetalProviderName` on line 78

Reply:
Thanks @aayushrangwala

Reply:
looks good to me.
/lgtm